### DISCUSSION TITLE
Some questions about STGCN

### DISCUSSION THREAD

**Discussion by lzm-bryan** [hazdzz/stgcn#25] [Q&A] [answered]
I would like to ask about the results of running STGCN on the PEMS-BAY dataset. When I reproduced the code, I found that the performance on the PEMS-BAY dataset was particularly good. It‘s much better than the results reported by others in papers. I wonder if this is normal. Thanks.

**Comment by hazdzz** [comment, marked as answer]
Thank you for paying attention to this repo. The experimental results are all normal, and the code is also open source. If you have any questions, feel free to inspect the code.
